src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ParameterAssignmentCheck.java

```diff
@@ -50,6 +50,29 @@
  * &lt;module name=&quot;ParameterAssignment&quot;/&gt;
  * </pre>
  * <p>
+ * Example :
+ * </p>
+ * <pre>
+ * class MyClass {
+ *   int method1(int parameter1, int parameter2) {
+ *     if (parameter2 > 50) {
+ *       parameter1 -= 2;  // violation, Parameter assignment is poor programming practice
```

Comment:
show simple type of assignment and simplify comment as it is explained in description of Check
```
 *       parameter2 = 2;  // violation
 *       parameter1 -= 2;  // violation
```

Reply:
done